src/core/Memory.test.ts

```diff
@@ -1,12 +1,23 @@
-import {getValues, saveValues} from "./Memory";
+import {getValues, saveValues, storageKey} from "./Memory";
+
+beforeEach(() => {
+    localStorage.clear();
+})
 
 test("Test That Value Retrieved will Be Empty If not Set", () => {
     expect(getValues()).toEqual([]);
 })
 
-test("Test that Value Can Be Saved And Retrieved", () => {
+test('Test That Value Will Be Saved In LocalStorage', () => {
     const value = [{date: new Date(), value: 21}, {date: new Date(), value: 48}];
     saveValues(value);
-    expect(getValues()).toEqual(value);
+    expect(localStorage.setItem).toHaveBeenCalledWith(storageKey, JSON.stringify(value));
```

Comment:
I don't think it should be tested that thoroughly.
One assert should be ok.
I created this test more like an acceptance criteria.
'saved and retrieved' is more like functional test.

Reply:
ok